<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(193, 197, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>695</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startMotor">
       <property name="text">
        <string>Pick-up RBR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fillVessel">
       <property name="text">
        <string>RBR to vessel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rbrToVesse">
       <property name="text">
        <string>Fill vessel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="leaveRbr">
       <property name="text">
        <string>Start motor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="liftRbr">
       <property name="text">
        <string>Stop motor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopMotor">
       <property name="text">
        <string>Lift RBR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pickUp">
       <property name="text">
        <string>Leave RBR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
